### ISSUE TITLE
Theme stylesheet gets pushed out even though it is priority 1

### ISSUE THREAD

**Issue by schlessera** [ampproject/amp-wp#3869]
## Bug Description

Whenever the CSS hits the 50KB limit, the theme stylesheet, being one of the most important ones, is being purged first, even though it is prioritized at priority 1.

## Expected Behaviour

The theme's stylesheet should be prioritized to only be dropped last, as it has the most impact on the visual similitude between AMP and non-AMP.

## Steps to reproduce

1. Activate the TwentyTwenty theme and activate it.
2. Install the Query Monitor plugin and activate it.
3. Go to a page with lots of diverse content.
4. **Verify:** The page looks visually broken as the theme stylesheet is not applied.
5. Open the "View Page Source..." page.
6. **Verify:** The CSS manifest comment at the top of the HTML source shows that the big Query Monitor stylesheet is kept while the theme stylesheet is dropped.

## Screenshots

![Image 2019-12-02 at 9 49 53 AM](https://user-images.githubusercontent.com/83631/69945278-86d0fc80-14e9-11ea-9081-577e7e1e1313.png)

---------------

_Do not alter or remove anything below. The following sections will be managed by moderators only._

## Acceptance criteria

* 

## Implementation brief

* 

## QA testing instructions

* 

## Demo

* 

## Changelog entry

*

**Comment by schlessera** [comment]
It seems the above is a different issue than what we discussed in Slack. In Slack we talked about the theme's inline styles not getting any prioritization.

The above issue is the regular stylesheet, though, which gets a priority of `1` but is still pushed out over multiple other ones with less priority.

The reason might be that the stylesheet is just so large it will never really fit. I'll dive in deeper.
